style.css

```diff
@@ -518,3 +538,222 @@ textarea {
   place-items: center;
   margin-top: 20px;
 }
+
+@media screen and (min-width: 768px) {
+  .cont-list {
+    display: flex;
+    gap: 55px;
+    margin-right: 45px;
+    list-style: none;
+    margin-left: 60%;
+  }
```

Comment:
- Hey, please, let's ensure the nav links do not show on the mobile version so we can still have the mobile version design unchanged. see the image below:

![Screenshot from 2023-02-23 22-33-41](https://user-images.githubusercontent.com/59397159/221036201-fe4b22cf-8e7a-4cdb-97bf-d49ebfb566ce.png)

Reply:
- Hey, after you've fixed the changes above, Please, ensure your mobile version design is not affected. The mobile design should be the same and not changed or affected by the desktop version. As of now, your mobile design is being affected, the design has changed. please, let's work on this.